Week1/queries.js

```diff
@@ -0,0 +1,98 @@
+const mysql = require('mysql');
+const connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'hyfuser',
+  password: 'hyfpassword',
+  database: 'world',
+});
+
+connection.connect(error => {
+  if (error) {
+    throw error;
+  }
+  console.log('Connected.');
+});
+
+const queries = {
+  createDatabase: `CREATE DATABASE IF NOT EXISTS world;`,
+
+  useDatabase: `USE worldData`,
+
+  createTableCountry: `CREATE TABLE IF NOT EXISTS country (code CHAR(3), Name CHAR(52), Continent CHAR (30), Region CHAR(26),SurfaceArea FLOAT(10,2),Indepyear SMALLINT(6),Population INT(11),LifeExpectancy FLOAT(3.1),GNP FLOAT(10.2),LocalName CHAR(45),GovernmentForm CHAR(45),HeadOfState CHAR(60),Capital INT(11),Code2 CHAR(2),PRIMARY KEY(Code));`,
+
+  createTableCity: `CREATE TABLE IF NOT EXISTS city(ID INT(11) AUTO_INCREMENT,Name CHAR(35),CountryCode CHAR(3),District CHAR(20),Population INT(11),PRIMARY KEY (ID),KEY CountryCode (CountryCode))`,
```

Comment:
Nice that you defined primary keys.
What type of key is CountryCode? What table and column does it refer to?

Reply:
I got that. I made a reference to countries(code).